definitions/install.yaml

```diff
@@ -469,6 +469,16 @@ metadata:
   name: kyverno-service-account
   namespace: kyverno
 ---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: policyviolation
```

Comment:
lets name this `kyverno:policyviolation` or `kyverno:policyviolations` to keep the resource names in sync.

Reply:
Also, update the documentation after the name changes.